clickhouse/metadata.csv

```diff
@@ -0,0 +1,202 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+clickhouse.background_pool.processing.task.active,gauge,,task,,"Number of active tasks in BackgroundProcessingPool (merges, mutations, fetches, or replication queue bookkeeping)",-1,clickhouse,
```

Comment:
For each description that starts with `Number of...`, change to `The number of..`